site/src/components/common/PostCard/PostCard.tsx

```diff
@@ -1,13 +1,9 @@
 import { memo, useMemo } from 'react'
 import cx from 'classnames'
-import { string, shape, bool, InferProps } from 'prop-types'
-
-import Icon from 'components/Icon'
-import Button from 'components/Button'
 
 import s from './PostCard.module.scss'
-import { getBlogPosts } from 'src/strapi'
 import { MappedBlog } from 'src/strapi/types/entities'
+import Card from 'components/Card'
```

Comment:
Now Card is not a default export

Reply:
https://github.com/NilFoundation/nil.foundation/actions/runs/6704650284/job/18217466130?pr=84

Reply:
Update that